recipes/molmod/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is unnecessary as conda-build will use the one from the source tarball.